ports/stm32/boards/stm32wbxx_hal_conf_base.h

```diff
@@ -64,19 +65,56 @@
 #define HAL_USART_MODULE_ENABLED
 
 // Oscillator values in Hz
-#define MSI_VALUE   (4000000)
+#define MSI_VALUE (4000000)
 
 // SysTick has the highest priority
 #define TICK_INT_PRIORITY (0x00)
 
 // Miscellaneous HAL settings
-#define DATA_CACHE_ENABLE           1
-#define INSTRUCTION_CACHE_ENABLE    1
-#define PREFETCH_ENABLE             0
-#define USE_SPI_CRC                 0
-#define USE_RTOS                    0
+#define DATA_CACHE_ENABLE 1
+#define INSTRUCTION_CACHE_ENABLE 1
+#define PREFETCH_ENABLE 0
+#define USE_SPI_CRC 0
+#define USE_RTOS 0
```

Comment:
best to leave these lines untouched

Reply:
I think it was codeformat.py that "cleaned" that up, I've fixed these and the new ones as suggested, see what CI here things of it :-)